docs/src/pages/index.tsx

```diff
@@ -1,25 +1,42 @@
 import React from "react";
 import { Link } from "gatsby";
-import { Heading, Inline } from "@kiwicom/orbit-components";
-import { StarEmpty } from "@kiwicom/orbit-components/icons";
-import styled, { css } from "styled-components";
+import { Heading, Inline, Stack } from "@kiwicom/orbit-components";
+import { StarEmpty, NewWindow } from "@kiwicom/orbit-components/icons";
+import { css } from "styled-components";
 import { WindowLocation } from "@reach/router";
 
 import ArrowRight from "../components/ArrowRight";
 import Layout from "../components/Layout";
 import RocketImage from "../components/RocketImage";
 import ButtonLink from "../components/ButtonLink";
 import Tile from "../components/Tile";
+import BrandedTile from "../components/BrandedTile";
+import GitHubLogo from "../images/github-full.svg";
+import SpectrumLogo from "../images/spectrum.svg";
+import TwitterLogo from "../images/twitter.svg";
+import srcTequila from "../images/tequila.png";
 
-const TileWrapper = styled.div`
-  flex: 1;
-  align-self: stretch;
-  display: flex;
-  flex-direction: column;
-  > :only-child {
-    flex: 1;
-  }
-`;
+function TileRow({ children }: { children: React.ReactNode }) {
+  return (
+    <Inline spacing="XLarge">
+      {React.Children.map(children, tile => (
+        <div
```

Comment:
Decided in Slack not to do it because `React.Children.map` already [does this](https://github.com/facebook/react/blob/4190a34588ec0b05aac6f050ec656ec4b3f005d7/packages/react/src/ReactChildren.js#L120).